### PR TITLE
signup

### PR DESCRIPTION
**Pull request by Atuheirebenadate**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
+
+
+  function App() {
+    return (
+      <div className="App">
+        {/* <Header/> */}
+       <Signup/>
+       hello
 
-function App() {
```
Comment (amyyalex) [inline src/App.js:4]: Why is line 4 to line 23 deleted?

File: src/componets/Signup.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                        </label>
+                </div>
+                <div className='txt-fld'>
+                        <label>Password
+                        <input type="password" name="password" placeholder="" />
+                        </label>
+                </div>
+
+                <div className='check_box'>
+                        {/* <label>
```
Comment (amyyalex) [inline src/componets/Signup.js]: Why is line 44 to 48 commented out?
I suggest you delete the lines if they don't have any function

File: src/componets/Signup.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                        </label> */}
+                       {/* <label className='forget'>forget Password?</label> */}
+                  </div>
+
+                <div>
+                        <button type="login">Sign in</button>
+                    </div>
+        
+
+                    <p>____________________________ or Signup with _____________________________</p>
```
Comment (amyyalex) [inline src/componets/Signup.js]: What is the essence of the undescore symbol used in line 56? 
If you were trying to acheive a space without line break, you should use &nbsp; instead
or if you trying to acheive the underscroe itself, you should use the html code for it which is &#95;

File: src/css/signup.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+ .container2{
+    background-color: black;
+    padding: 20px;
+    color: beige;
+    margin-left: 100%;
+    /* max-width: 100px; */
+    
+
+ }
+ /* .all{
```
Comment (amyyalex) [inline src/css/signup.css:72]: If the commented out lines dosen't serve any function, it's best removed so the code looks a bit tidy

File: src/header.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
Comment (amyyalex) [inline src/header.js]: Move this file into the components folder

**Review by amyyalex — CHANGES_REQUESTED** [review]
I've left a few comments that should be addressed before this gets merged.

**Comment by HassanBahati** [comment]
@amyyalex please review this PR and merge into develop

**Review by amyyalex — APPROVED** [review]

**Comment by HassanBahati** [comment]
@Atuheirebenadate please ensure to pull from the upstream, your code references very old changes leading to enormous conflicts.

**Pull request closed without merging**